contracts/script/multisigTransactionProposals/safeSDK/modifyProverModeProposal.ts

```diff
@@ -28,17 +32,24 @@ async function main() {
     const safeAddress = getEnvVar("SAFE_MULTISIG_ADDRESS");
     isValidEthereumAddress(safeAddress);
     const safeSdk = await Safe.create({ ethAdapter, safeAddress });
-
-    const permissionedProverAddress = getEnvVar("APPROVED_PROVER_ADDRESS");
-    isValidEthereumAddress(permissionedProverAddress);
-
-    await proposeSetProverTransaction(
-      safeSdk,
-      safeService,
-      await orchestratorSigner.getAddress(),
-      safeAddress,
-      permissionedProverAddress,
-    );
+    const orchestratorSignerAddress = await orchestratorSigner.getAddress();
+
+    if (command == SET_PROVER_CMD) {
```

Comment:
`===` is the prefered comparison operator in a JavaScript environment between JavaScript primitives.  Otherwise loose or truthy equality might get through, in this case it likely doesn't matter since we're comparing strings.  But I think, in general, it's better to err on the side of caution.

Reply:
oh good point, addressed here https://github.com/EspressoSystems/espresso-sequencer/commit/b9c694be95c90c23b91c79271db861cc9991c0cb